### PR TITLE
added after_login_hint for outlook & hotmail

### LINKED ISSUE #54: add hotmail.com, outlook.com, live.com
this was hardcoded until now, this was the code i just removed:
```
    private boolean isOutlook(String email) {
        return email != null
           && (email.toLowerCase().contains("@outlook.") || email.toLowerCase().contains("@hotmail."));
    }

    private void showOutlookHint()
    {
        ....
            new AlertDialog.Builder(this)
                .setMessage(
                      "Outlook- and Hotmail-e-mail-addresses "
                    + "may currently not work as expected "
                    + "as these servers may remove some important transport information."
                    + "\n\n"
                    + "Hopefully sooner or later there will be a fix; "
                    + "for now, we suggest to use another e-mail-address "
                    + "or try Delta Chat again when the issue is fixed.")
                .setPositiveButton(R.string.ok, null)
                .show();
    }
```

### PR DESCRIPTION
**Pull request by missytake**
closes #54

### CODE REVIEW

**Comment by missytake** [comment]
In this case, I didn't try it myself, but used these sources:

* https://support.mozilla.org/en-US/kb/thunderbird-and-hotmail
* https://www.serversettings.email/live.com-email-server-settings-imap.php
* oral tradition of the problems between DC & hotmail

File: _providers/hotmail.md [hunk 1]
```diff
@@ -6,13 +6,29 @@ domains:
...
+    hostname: smtp-mail.outlook.com
+    port: 587
 before_login_hint: |
   Outlook email addresses will not work as expected as these servers remove some important transport information.
   Hopefully sooner or later there will be a fix, for now we suggest to use another email address.
+after_login_hint: |
+  Outlook email addresses will not work as expected as these servers remove some important transport information.
+  Unencrypted 1-on-1 chats kind of work, but groups and encryption don't.
+  Hopefully sooner or later there will be a fix, for now we suggest to use another email address.
+  You can compare email providers at https://providers.delta.chat :)
```
Comment (pabzm) [inline _providers/hotmail.md]: I'd like to leave this line out for now. This data is going live soon, but providers.delta.chat currently doesn't show the state of any provider.

Additionally, providers.delta.chat isn't very helpful in actually comparing providers. It only shows an overview which providers work with Delta Chat — which is a fine and well-restricted purpose for a simple website, but not a comparison aid. So we should think with a different wording anyway, something like
> You may check in advance if your newly chosen provider works well with Delta Chat at https://providers.delta.chat/

Comment (missytake) [reply _providers/hotmail.md]: Sounds good. I'll fix the status issue, then it doesn't speak anything against guiding people to the overview site, right? Better than not offering it, I'd say.

Comment (pabzm) [reply _providers/hotmail.md]: The overview page is out of scope currently, let's speak about that when the current project is finished.

**Pull request merged**